spdm_dump/spdm_dump.h

```diff
@@ -33,6 +33,9 @@
 #include "stdlib.h"
 #include "string.h"
 
+/*the max session number spdm_dump can dump, this macro is a config*/
+#define LIBSPDM_MAX_SESSION_NUM 10
```

Comment:
why not just use the MACRO in config.h?
Why you need duplicate it?

Reply:
Sorry. I don't the  MACRO in libspdm_config.h before. I will change it.